resources/functions/user-management/cognito_user_management_service.py

```diff
@@ -142,4 +148,6 @@ def __init__(self, user_name=None, user_role=None,
         self.status = status
         self.enabled = enabled
         self.created = created
-        self.modified = modified
\ No newline at end of file
+        self.modified = modified
+    def __str__(self):
+        return jsonpickle.encode(self, unpicklable=False, use_decimal=True)
```

Comment:
what is this being used for?

Reply:
This is used to output to logs the response of cognito_user_management_service.

Reply:
Removed it with jsonpickle